.github/workflows/docs.yaml

```diff
@@ -61,13 +61,17 @@ jobs:
         run: make docs
 
       - uses: webfactory/ssh-agent@v0.6.0
-        if: github.event_name != 'pull_request'
+        env: 
+          DEPLOY_DOCS: ${{ secrets.DEPLOY_DOCS  }}
+        if: ${{ (github.event_name != 'pull_request') && (env.DEPLOY_DOCS != '') }} 
```

Comment:
I am concerned that if I put a secret in an environment variable and then use that environment variable elsewhere, that secret will show up. Wouldn't it be better to check the repository owner than that?